.travis.yml

```diff
@@ -11,8 +11,8 @@ python:
 
 env:
   - VERSION="8.0" LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0" INCLUDE="woocommerceerpconnect"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0" INCLUDE="woocommerceerpconnect"
+  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0" INCLUDE="connector_woocommerce"
```

Comment:
@dreispt : which i think is not good approach to test, as primary goal for testing would be to test integrity with dependent modules.